include/utils/math/geometry/rotation2d.h

```diff
@@ -305,6 +312,21 @@ double wrap_degrees_360(const double &angle);
  */
 double wrap_revolutions_360(const double &angle);
 
+/**
+ * General function for converting degrees to radians
+ * @param deg the angle in degrees
+ * @return the angle in radians
+ */
+double deg2rad(double deg);
+
+/**
+ * General function for converting radians to degrees
+ * @param r the angle in radians
+ * @return the angle in degrees
+
+ */
+double rad2deg(double r);
+
 /**
  * Calculates the mean of a list of angle values directly.
  * !! DOES NOT WRAP INPUTS (probably not useful) !!
```

Comment:
lmao @PascalSkylake  i saw this in the cpp file and was like hm this doesnt look very useful it doesnt wrap so it like doesnt do what you want it to do half the time. and then i scrolled down and saw this